nebari/initialize.py

```diff
@@ -49,14 +49,8 @@ def base_configuration():
                 "hub_title": None,
                 "hub_subtitle": None,
                 "welcome": None,
-                "logo": "/hub/custom/images/jupyter_nebari_logo.svg",
-                "primary_color": "#4f4173",
-                "secondary_color": "#957da6",
-                "accent_color": "#32C574",
-                "text_color": "#111111",
-                "h1_color": "#652e8e",
-                "h2_color": "#652e8e",
-                "version": f"v{__version__}",
+                "logo": "https://raw.githubusercontent.com/nebari-dev/nebari-design/main/logo-mark/horizontal/Nebari-Logo-Horizontal-Lockup-White-text.svg",
+                "version": f"{__version__}",
```

Comment:
@iameskild, you may drop the version field and switch it to `display_version.` The value is passed under `input_vars` automatically when the above is present, but there is no version assigned.

https://github.com/nebari-dev/nebari/blob/c08ee7164693ddd05f7956a932f7628bfee98ebc/qhub/stages/input_vars.py#L264